I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 12, in script call
    call Chap0
  File "game/script_S0.rpy", line 17, in script
    jump name_choose1
ScriptError: could not find label 'name_choose1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 12, in script call
    call Chap0
  File "game/script_S0.rpy", line 17, in script
    jump name_choose1
  File "renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'name_choose1'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Student Life 1.0
Tue May  3 17:22:42 2022
